### PR TITLE
use warnw instead of warnf

### PR DESCRIPTION
**Pull request by secmask**

### CODE REVIEW

File: common/archive/s3.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func (s *S3Archive) CheckFileIntergrity(bucketName string, awsfolderPath string,
 	file, err := os.Open(filePath)
 	defer func() {
 		if cErr := file.Close(); cErr != nil {
-			s.l.Warnf("File close error: %+v", cErr)
+			s.l.Warnw("File close", "err", cErr)
```
Comment (ducquangkstn) [inline common/archive/s3.go]: the msg shoud be `Failed to close file`

File: common/blockchain/transaction.go [hunk 2]
```diff
@@ -54,7 +54,7 @@ var errNotCached = errors.New("sender not cached")
 func setSenderFromServer(tx *types.Transaction, addr ethereum.Address, block ethereum.Hash) {
 	// Use types.Sender for side-effect to store our signer into the cache.
 	if _, err := types.Sender(&senderFromServer{addr, block}, tx); err != nil {
-		zap.S().Warnf("Type sender error: %+v", err)
+		zap.S().Warnw("Type sender", "err", err)
```
Comment (ducquangkstn) [inline common/blockchain/transaction.go]: The msg should be `Failed to get sender from signer and transaction`

File: core/reserve_core.go [hunk 3]
```diff
@@ -164,9 +164,10 @@ func (rc ReserveCore) Deposit(
 	}
 	recordActivity := func(status, txhex, txnonce, txprice string, err error) error {
 		uid := uidGenerator(txhex)
-		rc.l.Warnf(
-			"Core ----------> Deposit to %s: token: %s, amount: %s, timestamp: %d ==> Result: tx: %s, error: %s",
-			exchange.ID(), token.ID, amount.Text(10), timepoint, txhex, common.ErrorToString(err),
+		rc.l.Warnw(
```
Comment (ducquangkstn) [inline core/reserve_core.go]: Pls change it into level info

File: data/fetcher/fetcher.go [hunk 4]
```diff
@@ -740,7 +740,7 @@ func (f *Fetcher) FetchStatusFromExchange(exchange Exchange, pendings []common.A
 			// long time. We will just consider it as a failed activity.
 			timepoint, err1 := strconv.ParseUint(string(activity.Timestamp), 10, 64)
 			if err1 != nil {
-				f.l.Warnf("Activity %v has invalid timestamp. Just ignore it.", activity)
+				f.l.Warnw("Activity has invalid timestamp. Just ignore it.", "activity", activity)
```
Comment (ducquangkstn) [inline data/fetcher/fetcher.go]: we should log activity.Timestamp

File: data/storage/bolt.go [hunk 5]
```diff
@@ -791,12 +791,12 @@ func (bs *BoltStorage) GetAllRecords(fromTime, toTime uint64) ([]common.Activity
 func interfaceConverstionToUint64(l *zap.SugaredLogger, intf interface{}) uint64 {
 	numString, ok := intf.(string)
 	if !ok {
-		l.Warnf("(%v) can't be converted to type string", intf)
+		l.Warnw("can't be converted to type string", "value", intf)
```
Comment (ducquangkstn) [inline data/storage/bolt.go]: should we log the type of `intf`

File: data/storage/bolt.go [hunk 6]
```diff
@@ -791,12 +791,12 @@ func (bs *BoltStorage) GetAllRecords(fromTime, toTime uint64) ([]common.Activity
...
 		return 0
 	}
 	num, err := strconv.ParseUint(numString, 10, 64)
 	if err != nil {
-		l.Warnf("ERROR: parsing error %s, inteface conversion to uint64 will set to 0", err)
+		l.Warnw("ERROR: parsing error, interface conversion to uint64 will set to 0", "err", err)
```
Comment (ducquangkstn) [inline data/storage/bolt.go]: pls log `numString `

File: exchange/binance.go [hunk 7]
```diff
@@ -462,8 +462,7 @@ func (bn *Binance) FetchOnePairTradeHistory(
 	tokenPair := fmt.Sprintf("%s-%s", pair.Base.ID, pair.Quote.ID)
 	fromID, err := bn.storage.GetLastIDTradeHistory(tokenPair)
 	if err != nil {
-		bn.l.Warnf("Cannot get last ID trade history",
-			"err", err)
+		bn.l.Warnw("Cannot get last ID trade history", "err", err)
```
Comment (ducquangkstn) [inline exchange/binance.go]: pls add `tokenPair ` into log

File: common/archive/s3.go (hunk 1 above)
Comment (secmask) [reply common/archive/s3.go]: a line with "err = xxx" should also mean it failed.

File: data/storage/bolt.go (hunk 6 above)
Comment (secmask) [reply data/storage/bolt.go]: err message will has numString inside

File: data/fetcher/fetcher.go (hunk 4 above)
Comment (secmask) [reply data/fetcher/fetcher.go]: ok

File: core/reserve_core.go (hunk 3 above)
Comment (secmask) [reply core/reserve_core.go]: added if statement to check if err is not nil and use correspond level.

**Comment by ducquangkstn** [comment]
@secmask I have pushed several adjustment for warnning, pls check it

**Comment by secmask** [comment]
> @secmask I have pushed several adjustment for warnning, pls check it

looks ok to me.

**Pull request merged**